core/src/main/java/feast/core/config/WebSecurityConfig.java

```diff
@@ -37,10 +47,15 @@
    */
   @Override
   protected void configure(HttpSecurity http) throws Exception {
+    List<String> matchersToBypass = new ArrayList<>(List.of("/actuator/**", "/metrics/**"));
+
+    if (feastProperties.securityProperties().getAuthentication().isDisableRestControllerAuth()) {
```

Comment:
Should be `if (!feastProperties.securityProperties().getAuthentication().isDisableRestControllerAuth())` here.